proposed/code-of-conduct.md

```diff
@@ -0,0 +1,101 @@
+
+This includes the following:
+
+- The official MODX Forums, currently hosted at forums.modx.com
+- The “revolution” and “evolution” repositories hosted on GitHub.com, including the issues, pull requests, commit messages, wikis, and other communication that takes place there. 
+- The MODX Community Slack team (accessible through modxcommunity.slack.com and modx.org) 
+- Social Media channels managed or moderated by the Project Leadership or MODX Advisory Board, such as the LinkedIn MODX Group and official MODX Facebook page. 
+
+#### 1.3 Applicability of the Code of Conduct (Offline)
+
+The Code of Conduct is also to apply to offline channels, and in particular events that meet the following criteria:
+
+- The event is organised, hosted or managed by the MODX Project Leadership, the MODX Advisory Board, or a MODX Ambassador, or if its name implies it is. For example a “MODXpo”, “MODX Day”, or “MODX Weekend” must use the Code of Conduct even if it is organised, hosted or managed by a third party. 
+- The event carries a focus on MODX.
+
+For offline events that meet the criteria, the organiser(s), host(s) or manager(s) of the event must ensure:
+
+1. The code of conduct is easily accessible for (prospective) attendees.
+2. Before and during the event attendees are prompted to the existence of the code of conduct.  
+3. For events with over 50 attendees at least one person must be available as an on-site code of conduct liaison. This person must understand the Code of Conduct and the response guidelines, and be able of acting on them should an incident occur during the event. Should an incident occur, the liaison is to report this as soon as possible. It must be clear to the attendees how the liaison can be contacted immediately during the event, for example by publishing a phone number on the website, on the event program, and/or by announcing it during the start of the event.
+4. For events with over 150 attendees, there must be at least three liaisons available. 
+
+#### 1.4 Individual Representation
+
+When individuals represent the MODX project in other channels, they are also expected to follow the MODX Code of Conduct. This includes for example when speaking about MODX at a meetup or usergroup not focused on MODX, or on online channels that the MODX Project Leadership, MODX Advisory Board or MODX Ambassadors hold no direct control over. 
+
+### 2. Incident Response
+
+To report a breach of the Code of Conduct a special email address is available. Contacting this email address will notify the MODX Project Leadership, the MODX Advisory Board Chair, the MODX Advisory Board Vice-Chair and one or more appointed and publicly listed members of the MODX Advisory Board or outside experts. 
```

Comment:
Project Leadership needs to make this email address available. 

Recommendation-wise I'm thinking of adding a separate section that describes who is in the group of responders (and why) and what rules they have to abide by, like the confidentially in the next paragraph.